mods/taskbar-clock-customization.wh.cpp

```diff
@@ -686,7 +714,8 @@ void LoadSettings()
     g_settings.webContentsStart = Wh_GetStringSetting(L"WebContentsStart");
     g_settings.webContentsEnd = Wh_GetStringSetting(L"WebContentsEnd");
     g_settings.webContentsMaxLength = Wh_GetIntSetting(L"WebContentsMaxLength");
-    g_settings.webContentsUpdateInterval = Wh_GetIntSetting(L"WebContentsUpdateInterval");
+    g_settings.webContentsUpdateInterval = std::stof(Wh_GetStringSetting(L"WebContentsUpdateInterval"));
```

Comment:
You have a memory leak here, the string returned by `Wh_GetStringSetting` is never freed.